index.html

```diff
@@ -2059,11 +2060,27 @@ <h2>Play all of your retro games on the go</h2>
             <img src="img/emulators.jpg" />
           </div>
           <div class="col_6">
-            <h2>All your favourite systems</h2>
+            <h2>All of your favourite emulators</h2>
             <p class="large">
-              Atari, Genesis/Mega Drive , Sega CD, Sega 32X, PC Engine, NES, Super Nntendo , MAME, FinalBurn Neo, Master
-              System, Game Boy, Neo Geo Pocket, Game Gear, Nintendo DS, Nintendo 3DS, Sony PSP, Dreamcast, Playstation,
-              Nintendo 64, Wii, GameCube, Wii U, Switch, etc.
+            <ul>
```

Comment:
@rawdatafeel I prefer to keep this clean, please remove the ul and left the previous list :)

Reply:
Moved to a credits section, let me know your thoughts on that.